index.html

```diff
@@ -0,0 +1,350 @@
+          <figure>
+            <img src="/img/jane.jpg" alt="Picture of Jane Ella Doe" title="Jane Ella Doe" class="profile-image circle">
+            <figcaption>
+              <a href="https://www.facebook.com" class="social-icons icon-facebook"></a>
+              <a href="https://twitter.com" class="social-icons icon-twitter"></a>
+              <a href="https://dribbble.com" class="social-icons icon-dribbble"></a>
+              <a href="https://www.linkedin.com" class="social-icons icon-linkedin2"></a>
+              <a href="https://plus.google.com" class="social-icons icon-google-plus"></a>
+              <p class="team-text profile-text">Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard.</p>
+            </figcaption>
+          </figure>
+        </section>
+        <section class="team-profile">
+          <h3 class="team-header profile-header">JESSIE DOE</h3>
+          <h4 class="team-text profile-text">Business and legacy</h4>
+          <figure>
+            <img src="/img/jessie.jpg" alt="Picture of Jessie Doe" title="Jessie Doe" class="profile-image circle">
+            <figcaption>
+              <a href="https://www.facebook.com" class="social-icons icon-facebook"></a>
+              <a href="https://twitter.com" class="social-icons icon-twitter"></a>
+              <a href="https://dribbble.com" class="social-icons icon-dribbble"></a>
+              <a href="https://www.linkedin.com" class="social-icons icon-linkedin2"></a>
+              <a href="https://plus.google.com" class="social-icons icon-google-plus"></a>
+              <p class="team-text profile-text">Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard.</p>
+            </figcaption>
+          </figure>
+        </section>
+        <section class="clients">
+          <h3 class="client-section-header gray-line">CLIENTS</h3>
+          <button class="icon-chevron-left client-button circle"></button>
+          <ul>
+            <li>
+              <figure class="client-figure">
+                <img src="/img/broderic.jpg" alt="Image of Matthew Broderic" title="Matthew Broderic" class="client-image circle">
+                <figcaption class="client-caption">
+                  <h4 class="team-header client-header"><span class="client-name">Matthew Broderic</span> (CEO Hopicola Inc.)</h4>
+                  <p class="team-text">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard my text ever since the 1500s, when lorem upsum is the best text for you.</p>
+                </figcaption>
+              </figure>
+            </li>
+          </ul>
+          <button class="icon-chevron-right client-button circle"></button>
```

Comment:
Please add a `type` attribute to the `button` element. Here it'd be `<button type="button">`.

Semantically, though, this should probably be an anchor. You're not inside of a form, and while clicking this link _does_ trigger an action (once the scrolling behavior is added), it more realistically functions like a link where it's taking the user to a specific piece of content.